_posts/2020-01-23-operating-system-and-browser-accessibility-display-modes.md

```diff
@@ -0,0 +1,373 @@
+
+This is helpful for:
+
+- People who want to reduce eye strain in a low or no light environment,
+- and/or who have certain [low vision](https://webaim.org/articles/visual/lowvision) or photosensitive conditions.
+
+An example of one of these conditions could be someone avoiding triggering a migraine by using Inverted Colors Mode to turn a brightly-colored application they need to use for work dark.
+
+### What it affects
+
+Inverted Colors mode affects **all** content displayed on the screen: Operating System UI, first and third party apps, and web content.
+
+<figure role="figure" aria-label="Safari will automatically invert images when Inverted Color Mode is activate.">
+  <img alt="A side-by-side comparison of regular and Inverted Colors Mode. The left screenshot is a Washington Post review of a Hokusai exhibit, including a headline, feature image of Mount Fuji, author photo and intro paragraph. The article’s design uses black text on a white background. The right screenshot is the same article, but with Inverted Colors Mode activated. The article is now white text on a black background." src="/img/posts/2020-01-23-operating-system-and-browser-accessibility-display-modes/inverted-colors.png">
+  <figcaption>Safari will automatically invert images when Inverted Color Mode is activate.</figcaption>
+</figure>
+
+#### How to enable it
+
+Assumes the latest version of each Operating System or browser.
+
+#### Android
+
+1. Go to **Settings**.
+1. Go to **System**.
+1. Go to **Accessibility**.
+1. Set the **Color Inversion** toggle to the **on** position.
+
+#### iOS
+
+1. Go to **Settings**.
+1. Go to **Accessibility**.
+1. Go to **Display & Text Size**.
+1. Set the **Smart Invert** or **Classic Invert** toggle to the **on** position.
+
+#### macOS
+
+1. Go to **System Preferences**.
+1. Go to **Accessibility**.
+1. Go to **Display**.
+1. Check **Invert colors**.
+
+#### Windows
+
+1. Go to **Settings**.
+1. Go to **Ease of Access**.
+1. Go to **Magnifier**.
+1. Set the **Invert colors** toggle to the **on** position.
+1. Set the **Magnifier** toggle to the **on** position.
+
+### How to target it in code
+
+You can use the [`inverted-colors` media feature](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/inverted-colors) to to target Dark Mode in CSS. In the following example, we invert images and video when Inverted Colors mode is active. This keeps them looking the way they were intended, while allowing the rest of the UI to change:
+
+``` css
+@media (inverted-colors: inverted) {
+  img,
+  video {
+    filter: invert(100%);
+  }
+}
+```
+
+## Reader Mode
+
+This mode is a feature found in most web browsers. When activated, it takes the main part of a web page and converts it into a format that is easier to read.
```

Comment:
...it takes the content identified as the primary content of a web page...


just want to shy away from saying "main part"  because content within `main` could very well not trigger reader mode.

Reply:
Good call. Updated.